README.md

```diff
@@ -1,43 +1,49 @@
-# An easy-to-use command line interface for Tastyworks!
+# tastyworks-cli
 
-## Installation
+An easy-to-use command line interface for Tastyworks!
 
-Currently unavailable pending release:
+## Installation
 
 ```
 $ pip install tastyworks-cli
 ```
 
-## Development
+## Usage
```

Comment:
users need more specific instruction on what csv file is required for the portfolio history functions e.g. `tw plot`. this might constitute a standalone section in the readme. 

it isn't necessarily obvious where this csv file should come from, so I recommend just stating "the csv file can be exported from the TW desktop app from the xyz tab", something of that nature. 

another thing - are there any other specifications on the csv file, i.e. what happens the file is exported with a starting date that splits a chain? 

so basically the readme just needs a description of the csv file required for `tw plot` input etc.